source/Ship.cpp

```diff
@@ -1959,7 +1962,8 @@ void Ship::Move(vector<Visual> &visuals, list<shared_ptr<Flotsam>> &flotsam)
 	if(acceleration)
 	{
 		acceleration *= slowMultiplier;
-		Point dragAcceleration = acceleration - velocity * (attributes.Get("drag") / mass);
+		double trueDrag = Drag();
+		Point dragAcceleration = acceleration - velocity * (trueDrag / mass);
```

Comment:
same here:
```suggestion
		Point dragAcceleration = acceleration - velocity * (Drag() / mass);
```
no need to define an extra variable